examples/dedicated_vmseries_and_autoscale/main.tf

```diff
@@ -11,7 +11,8 @@ resource "random_password" "this" {
 }
 
 locals {
-  vmseries_password = coalesce(var.vmseries_password, try(random_password.this[0].result, null))
+  vmseries_password               = coalesce(var.vmseries_password, try(random_password.this[0].result, null))
+  disable_password_authentication = local.vmseries_password == null ? true : false
```

Comment:
what do we need this one for?

Reply:
That solves the issue in which checkov status that you must disable password auth for more security. Putting it under a condition eliminates the scan basically (because it's dependent on the input).